src/coap_openssl.c

```diff
@@ -638,6 +638,19 @@ static long coap_sock_ctrl(BIO *a, int cmd, long num, void *ptr) {
 }
 #endif /* !COAP_DISABLE_TCP */
 
+static void coap_set_user_prefs(SSL_CTX *ctx) {
```

Comment:
We would need to deal with quotes here.
And there is a typo: `CLAGS` → `CFLAGS`

Reply:
Example now corrected.

Reply:
Fixed. PTAL.

Reply:
@biodranik Sorry to mess you around.  It has been pointed out that CPPFLAGS should be used instead of CFLAGS in the documentation as this is are pre-processor symbols.

Reply:
Fixed CFLAGS=>CPPFLAGS in documentation comment. PTAL.